ios-target-app/Home/Views/SaveTargetBottomSheetViewController.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  SaveTargetBottomSheetViewController.swift
+//  ios-target-app
+//
+//  Created by Raul Piñeres Carrera on 25/10/22.
+//
+
+import UIKit
+
+class SaveTargetBottomSheetViewController: UIViewController {
+  
+  private lazy var areaLabel = UILabel(
+    text: "home_area_label".localized,
+    size: .normal
+  )
+  
+  private lazy var areaField = UITextField(
+    target: self,
+    placeholder: "home_area_placeholder".localized
+  )
+  
+  private lazy var targetTitleLabel = UILabel(
+    text: "home_target_label".localized,
+    size: .normal
+  )
+  
+  private lazy var targetTitleField = UITextField(
+    target: self,
+    placeholder: "home_target_placeholder".localized
+  )
+  
+  private lazy var topicLabel = UILabel(
+    text: "home_topic_label".localized,
+    size: .normal
+  )
+  
+  private lazy var topicField = UITextField(
+    target: self,
+    placeholder: "home_topic_placeholder".localized
+  )
+  
+  private lazy var saveButton = UIButton(
+    text: "home_save_button_label".localized,
+    target: self
+  )
+  
+  override func viewDidLoad() {
+    super.viewDidLoad()
+    
+    setBottomSheetUIConfigs()
+    configureViews()
+  }
+  
+  func setBottomSheetUIConfigs() {
+    view.layer.cornerRadius = UI.BottomSheet.cornerRadius
+    view.layer.maskedCorners = [.layerMinXMinYCorner, .layerMaxXMinYCorner]
+    view.layer.shadowColor = UIColor.black.cgColor
+    view.layer.shadowRadius = UI.BottomSheet.shadowRadius
```

Comment:
I would create helper functions to add round corners and shadows to any `UIView` so you stay DRY